examples/model_selection/plot_roc.py

```diff
@@ -437,7 +439,19 @@
 # global performance of a classifier can still be summarized via a given
 # averaging strategy.
 #
-# Micro-averaged OvR ROC is dominated by the more frequent class, since the
-# counts are pooled. The macro-averaged alternative better reflects the
-# statistics of the less frequent classes, and then is more appropriate when
-# performance on all the classes is deemed equally important.
+# When dealing with imbalanced datasets, choosing the correct metric is crucial.
+# Accuracy may not always reflect model performance well in these cases.
+# For example, metrics such as F1 score, precision, or recall might be more
+# appropriate, depending on the business context or problem you are addressing.
```

Comment:
In the original issue we refer to business metric and not usual statistical metric. We could also call those utility metrics.

So we should state anything about precision and recall.

Reply:
Just to clarify are you suggesting to remove these specific mention of precision and recall and just write it as a general like `business/utility metric of importance based on use case` ?

Reply:
Yes sorry my sentence was not clear. I was indeed proposing to remove about the precision and recall.

Reply:
Thanks for clarifying, I'll make the changes and notify you for review.